templates/html/doxygen.css

```diff
@@ -228,6 +228,22 @@ a.codeRef, a.codeRef:visited, a.lineRef, a.lineRef:visited {
 	color: ##60; 
 }
 
+a.code.highlightclass {
+    /* style for links to class names in code snippets */
+}
+a.code.highlightfile {
+    /* style for links to file names in code snippets */
+}
+a.code.highlightnamespace {
+    /* style for links to namespace names in code snippets */
+}
+a.code.highlightmember {
+    /* style for links to member variable/function names in code snippets */
+}
+a.code.highlightconcept {
+    /* style for links to concept names in code snippets */
+}
```

Comment:
Just 5 ?

Reply:
I left out the others because I don't think they are ever produced in code snippets, or is that incorrect? Can a Doxygen “group” (TypeGroup) ever be linked to in a code snippet? Maybe `TypePackage` could occur in Java or Python code?

I'll add them just in case, if you think that's best :)